evm/src/main/java/org/hyperledger/besu/evm/worldstate/AbstractWorldUpdater.java

```diff
@@ -87,28 +89,28 @@ public MutableAccount createAccount(final Address address, final long nonce, fin
     final UpdateTrackingAccount<A> account = new UpdateTrackingAccount<>(address);
     account.setNonce(nonce);
     account.setBalance(balance);
-    return track(account);
+    return authorizedCodeService.processMutableAccount(this, track(account), address);
```

Comment:
have you accounted for DiffBasedWorldStateUpdater overrides of createAccount() , get() , getAccount() ?

It isn't obvious to me that the evm is always going to use instances that do not override these methods

Reply:
It was missing in `DiffBasedWorldStateUpdateAccumulator.createAccount`, I've added it. For devnet-2 we should be able to remove those from all the WorldUpdaters